src/lib.rs

```diff
@@ -315,15 +317,26 @@ impl PrometheusMetricsBuilder {
         self
     }
 
+    /// Report HTTP version of served request as "http_version" label
+    pub fn enable_http_version_label(mut self, enable: bool) -> Self {
+        self.enable_http_version_label = enable;
+        self
+    }
+
     /// Instantiate PrometheusMetrics struct
     pub fn build(self) -> Result<PrometheusMetrics, Box<dyn std::error::Error + Send + Sync>> {
         let http_requests_total_opts =
             Opts::new("http_requests_total", "Total number of HTTP requests")
                 .namespace(&self.namespace)
                 .const_labels(self.const_labels.clone());
 
-        let http_requests_total =
-            IntCounterVec::new(http_requests_total_opts, &["endpoint", "method", "status"])?;
+        let label_names = if self.enable_http_version_label {
+            ["http_version", "endpoint", "method", "status"].as_slice()
```

Comment:
I think this should be `version` to be consistent.

Reply:
I could rename, but on the other side I think it might possibly conflict with labels potentially added by scrapers. The name 'version' might be also confused with other things which might have version, like API version or something else. Maybe then I should change an API to `enable_http_version_label(label: String)` when user of library can define the label? Of course in that case it necessary to validate that chosen label is not among already used by library. What do you think?

Reply:
I think it should be the other way around. Folks adding labels should give it a `scraper_version` if they need to.

Reply:
Changed label name from `http_version` to just `version`.

Reply:
Changed label name from `http_version` to `version`.